src/util/SqlTagUtils.ts

```diff
@@ -0,0 +1,49 @@
+import { Driver } from "../driver/Driver"
+
+export function buildSqlTag(params: {
+    driver: Driver
+    strings: TemplateStringsArray
+    expressions: unknown[]
+}): { query: string; variables: unknown[] } {
+    let query = ""
+    const variables: unknown[] = []
```

Comment:
Elsewhere in the codebase we use the term "parameters" rather than "variables" for these placeholders in SQL queries. So renaming this (and the returned property) would be better for consistency.

Reply:
My bad, fixed in DataSource and EntityManager as well.